api/Window.json

```diff
@@ -3333,6 +3369,42 @@
               "deprecated": true
             }
           }
+        },
+        "relative-multi-screen": {
+          "__compat": {
+            "description": "Opened relative to the <a href='https://developer.mozilla.org/docs/Web/API/Window_Management_API/Multi-screen_origin'>Multi-screen origin</a>",
+            "support": {
+              "chrome": {
+                "version_added": "100"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "1",
+                "partial_implementation": true,
+                "notes": "Behavior is inconsistent/unreliable."
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false,
+                "notes": "Opened relative to the origin of the primary screen."
```

Comment:
Crikey, the Safari comments are not very encouraging.

I'm not certain it is really worth trying to understand and document the exact behavior of Safari, as it is so buggy and weird. For the moment I've just put the "Behavior is inconsistent/unreliable" note on the `Window` features for Safari, and left it as support `false` on the `Screen` features (because the original data said `false` for Safari, before I started working on this API).

Reply:
Thanks again for all your time and effort spent helping me to understand this stuff.